examples/parachain-example/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+//! # Tutorial: Subxt setup to interact with custom chains
+//!
+//! There are many parachains on Polkadot and Kusama. Using Subxt to connect to a parachain
+//! (or any other custom, substrate based chain) requires 2 steps:
```

Comment:
```suggestion
//! (or any other custom, substrate based chain) typically involves 2 steps:
```

(just because you can use the dynamic interface and not do step 1 at all if you don't want to, so it's not a requirement)